recipes/xarray_leaflet/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "xarray_leaflet" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "5f68b61633392404830c6abebe5f31733a87a25bb3eb5cb10d1b1b2491dec38f" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: {{ build }}
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5.0
+    - pip
+  run:
+    - python
+    - xarray
+    - ipyleaflet
+    - pillow
+    - rasterio
+    - affine
+    - mercantile
+
+test:
```

Always good to check the dependencies are fulfilled.
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

---

Thanks!